src/lemlib/logger/telemetrySink.cpp

```diff
@@ -0,0 +1,10 @@
+#include "telemetrySink.hpp"
+
+#include "fmt/format.h"
+#include "stdout.hpp"
```

Comment:
Clean up file includes
```suggestion
#include "fmt/format.h"
#include "lemlib/logger/telemetrySink.hpp"
#include "lemlib/logger/stdout.hpp"
```